docsets/Hapi/docset.json

```diff
@@ -1,12 +1,32 @@
 {
-    "name": "Hapi",
-    "version": "13.4",
-    "archive": "hapi-13.4.x.tgz",
-    "author": {
-        "name": "Timo Behrmann",
-        "link": "https://twitter.com/z0mt3c"
+  "name": "Hapi",
+  "version": "16.6",
+  "archive": "hapi.tgz",
+  "author": {
+    "name": "Timo Behrmann",
+    "link": "https://twitter.com/z0mt3c"
+  },
+  "aliases": [
+    "hapi",
+    "hapijs",
+    "hapi.js"
+  ],
+  "specific_versions": [
+    {
+      "version": "16.6",
+      "archive": "versions/16.6/hapi.tgz"
     },
-    "aliases": ["hapi",
-                "hapijs",
-                "hapi.js"]
+    {
+      "version": "15.2",
+      "archive": "versions/15.2/hapi.tgz"
+    },
+    {
+      "version": "14.2",
+      "archive": "versions/14.2hapi.tgz"
+    },
+    {
+      "version": "13.6",
+      "archive": "versions/13.6/hapi.tgz"
```

Comment:
You didn't add this archive. I think you added `versions/13.4/hapi.tgz` instead.